Include/XA/czyz/atc.atc

```diff
@@ -1,14 +1,12 @@
+// CENTER SECTORS
 CZYZ_CTR;125.775;
-CYAM_TWR;118.800;
-CYAM_GND;121.700;
-CYHM_TWR;125.000;
-CYHM_GND;121.600;
-CYKF_TWR;126.000;
-CYKF_GND;121.800;
-CYKZ_TWR;124.800;
-CYKZ_GND;121.800;
-CYXU_TWR;119.400;
-CYXU_GND;121.900;
+CZYZ_CE_CTR;125.775;
+CZYZ_PI_CTR;124.675;
+CZYZ_XI_CTR;121.225;
+CZYZ_SH_CTR;134.425;

```

Comment:
Just double checking (though I'm sure you've already thought of this), have these sectors been added to the database? If not, that needs to be done before we merge a change with un-connectable positions.

Reply:
They have been added to the database.